<h1>Individual Planning Report</h1>


<h2>1. Data Description:</h2>

In [4]:
library(tidyverse)

── Attaching core tidyverse packages ──────────────────────── tidyverse 2.0.0 ──
✔ dplyr     1.1.4     ✔ readr     2.1.5
✔ forcats   1.0.0     ✔ stringr   1.5.1
✔ ggplot2   3.5.1     ✔ tibble    3.2.1
✔ lubridate 1.9.3     ✔ tidyr     1.3.1
✔ purrr     1.0.2     
── Conflicts ────────────────────────────────────────── tidyverse_conflicts() ──
✖ dplyr::filter() masks stats::filter()
✖ dplyr::lag()    masks stats::lag()
ℹ Use the conflicted package (<http://conflicted.r-lib.org/>) to force all conflicts to become errors


In [3]:
players <- read_csv("IndividualGroupProject/players.csv")                

ERROR: Error in read_csv("IndividualGroupProject/players.csv"): could not find function "read_csv"


In [7]:
sessions <- read_csv("IndividualGroupProject/sessions.csv")

Rows: 1535 Columns: 5
── Column specification ────────────────────────────────────────────────────────
Delimiter: ","
chr (3): hashedEmail, start_time, end_time
dbl (2): original_start_time, original_end_time

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.


<p>There are two datasets to examine as part of the project. The first dataset "players.csv" contains player-specific information and cumulative play data, with 196 rows and 7 columns. The second dataset is "sessions.csv" which contains information about each individual session played, with 1535 rows and 5 columns.</p>

<h2>players.csv</h2>

<h3>Variables/Columns</h3>
<ol>
    <li><u>experience (chr):</u> A categorical variable used to report the player's experience level ("Beginner", "Amateur", "Regular", "Veteran", "Pro").</li>
    <li><u>subscribe (lgl):</u> A boolean categorical variable used to represent whether the players are subscribed or not subscribed.</li>
    <li><u>hashedEmail (chr):</u> A categorical variable used to recognize each player. Each player has a unique code to identify them by.</li>
    <li><u>played_hours (dbl):</u> A numerical variable used to report the player's total hours played.</li>
    <li><u>name (chr):</u> A categorical variable used to identify the player's name.</li>
    <li><u>Age (dbl):</u> A numerical variable used to report the player's age.</li>
</ol>

<h2>sessions.csv</h2>

<h3>Variables/Columns</h3>
<ol>
    <li><u>hashedEmail (chr):</u> A categorical variable used to recognize each player. Each player has a unique code to identify them by.</li>
    <li><u>start_time (chr):</u> The date and time of each session start stored as a character.</li>
    <li><u>end_time (chr):</u> The date and time of each session end stored as a character.</li>
    <li><u>original_start_time (dbl):</u> A numerical variable used as a Unix time identifier to represent the start time.</li>
    <li><u>original_end_time (dbl):</u> A numerical variable used as a Unix time identifier to represent the end time.</li>
</ol>
